lib/kv-redis.js

```diff
@@ -357,14 +370,28 @@ function(modelName, filter, options, callback) {
   function fetchFromRedis(cb) {
     if (cursor === null) cursor = '0';
 
-    ModelKeyComposer.compose(modelName, keyPattern, function(err, pattern) {
-      if (err) return cb(err);
-      self.execute('SCAN', [cursor, 'MATCH', pattern], function(err, result) {
+    ModelKeyComposer.compose(customModelName, keyPattern,
+      function(err, pattern) {
```

Comment:
While you are touching this code, have you considered switching to `=>` style? Will it allow us to keep the same level of indentation?

```js
ModelKeyComposer.compose(customModelName, keyPattern, (err, pattern) => {
  // ...
  this.execute('SCAN', [cursor, 'MATCH', pattern], (err, result) => {
    // ...
  });
});
```

Reply:
Is not possible, the `this` bind to different scope